### PR TITLE
[WIP] Goon Botany

### PR DESCRIPTION
**Pull request by Iamgoofball**
will finish someday(tm)

doesnt compile cause 420 blaze it

### CODE REVIEW

**Comment by ZomgPonies** [comment]
The sooner we get this done, the better off we'll be.

File: code/modules/hydroponics/goon/herb_seeds.dm [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+/datum/seed/asomna/robust
+	name = "robustasomna"
+	seed_name = "robust asomna"
+	display_name = "robust asomna"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/robustasomna)
+	mutants = null
+	packet_icon = "seed-robustasomna"
+	plant_icon = "robustasomna"
+	chems = list("hyperzine" = list(1,10))
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/goon/herb_seeds.dm]: Meth

File: code/modules/hydroponics/goon/herb_seeds.dm [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+/datum/seed/nureous
+	name = "nureous"
+	seed_name = "nureous"
+	display_name = "nureous"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/nureous)
+	mutants = list("fuzzynureous")
+	packet_icon = "seed-nureous"
+	plant_icon = "nureous"
+	harvest_repeat = 1
+	chems = list("hyronalin" = list(1,10))
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/goon/herb_seeds.dm]: poatssium iodide.

File: code/modules/hydroponics/goon/herb_seeds.dm [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+/datum/seed/commol
+	name = "commol"
+	seed_name = "commol"
+	display_name = "commol"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/commol)
+	mutants = list("burningcommol")
+	packet_icon = "seed-commol"
+	plant_icon = "commol"
+	harvest_repeat = 1
+	chems = list("kelotane" = list(1,10))
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/goon/herb_seeds.dm]: silver sulfa

File: code/modules/hydroponics/goon/herb_seeds.dm [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+
+/datum/seed/venne/curative
+	name = "curativevenne"
+	seed_name = "sunrise venne"
+	display_name = "sunrise venne"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/curativevenne)
+	mutants = null
+	packet_icon = "seed-curativevenne"
+	plant_icon = "curativevenne"
+	chems = list("oculine" = list(1,10), "mannitol" = list(1,10), "ryetalyn" = list(1,10))
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/goon/herb_seeds.dm]: mutadone instead of ryetalyn

File: code/modules/hydroponics/goon/herb_seeds.dm [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+/datum/seed/cannabis
+	name = "cannabis"
+	seed_name = "cannabis"
+	display_name = "cannabis"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/cannabis)
+	mutants = list("omegaweed", "lifeweed", "deathweed", "rainbowweed")
+	packet_icon = "seed-cannabis"
+	plant_icon = "cannabis"
+	harvest_repeat = 1
+	chems = list("spacedrugs" = list(1,10))
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/goon/herb_seeds.dm]: THC, soon to be an actual reagent!

File: code/modules/hydroponics/goon/weed_seeds.dm [hunk 6]
```diff
@@ -0,0 +1,30 @@
+/datum/seed/slurrypod
+	name = "slurrypod"
+	seed_name = "slurrypod"
+	display_name = "slurrypod"
+	mutants = list("omegaslurrypod")
+	packet_icon = "seed-slurrypod"
+	plant_icon = "slurrypod"
+	chems = list("toxin" = list(1,10))
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/goon/weed_seeds.dm]: You can actually use Toxic Slurry for this and the chem_splash var now---additionally you can use glowing slurry in place of "omega_mutagen".

**Comment by ZomgPonies** [comment]
So is this still being worked on?

**Comment by Iamgoofball** [comment]
i'm gonna port the final few plants we don't already have and then we have the problem of we need like 50+ sprites

**Comment by CrAzYPiLoT-SS13** [comment]
I can make some if they are not too complex, just don't expect a
masterpiece.

File: code/modules/hydroponics/goon/herb_seeds.dm [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+	name = "omegaweed"
+	seed_name = "omega weed"
+	display_name = "omega weed"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/omegaweed)
+	mutants = null
+	packet_icon = "seed-omegaweed"
+	plant_icon = "omegaweed"
+	chems = list("LSD" = list(1,10),"suicider" = list(1,10),"space_drugs" = list(1,10),"mercury" = list(1,10),"lithium" = list(1,10),
+	"atropine" = list(1,10), "ephedrine" = list(1,10), "impedrezene" = list(1,10),"methamphetamine" = list(1,10),"THC" = list(1,10),"capsaicin" = list(1,10),"psilocybin" = list(1,10),"hairgrownium" = list(1,10),
+	"ectoplasm" = list(1,10),"bathsalts" = list(1,10),"itching_powder" = list(1,10),"crank" = list(1,10),"krokodil" = list(1,10),"catdrugs" = list(1,10),"histamine" = list(1,10))
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/goon/herb_seeds.dm]: LSD = mindbreaker (for now until we have reagent decals and add in space fungsus on walls)
impedrezene = haloperidol.

Don't have ectoplasm or cat drugs yet.

File: code/modules/hydroponics/goon/grown_food.dm [hunk 8]
```diff
@@ -0,0 +1,184 @@
+/obj/item/weapon/reagent_containers/food/snacks/grown/tomatoexplosive
+	name = "seething tomato"
+	desc = "I say to-mah-to, you say KABOOM."
+	icon_state = "explosivetomato"
+	filling_color = "#FF0000"
+	potency = 10
+	plantname = "explosivetomato"
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/tomatoexplosive/throw_impact(atom/hit_atom)
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/goon/grown_food.dm]: would still probably be best if these functioned like primed grenades with random fuses--especially considering how easy these will be to produce.

**Comment by Midaychi** [comment]
Is it possible to add goone botany without replacing bay botany? Then you could have goone botany in hydro and bring back xenobotany for science, best of both worlds.

**Comment by Fox-McCloud** [comment]
Xenobotany isn't coming back to science--it concentrates way too much power in a singular department. Sorry.

**Comment by Spacemanspark** [comment]
Laaaaaame.

**Comment by ZomgPonies** [comment]
Do something to this or close the PR. Shit shouldn't take a month.

**Comment by Midaychi** [comment]
Rip weed

**Comment by Iamgoofball** [comment]
rip

**Pull request closed without merging**